Library.Tests/CalculationHelperTests.cs

```diff
@@ -0,0 +1,113 @@
+            (int a, int b, int expected)
+        {
+            int actual = CalculationHelper.GetCalcutaionResults(a, b);
+            Assert.AreEqual(expected, actual);
+        }
+
+        [TestCase(1, 2, 1)]
+        [TestCase(8, -3, 4)]
+        [TestCase(-10, 5, 2)]
+        [TestCase(-10, -3, 3)]
+        public void DetermineQuaterOfEnteredCoords_WhenXYNotZero_ShouldGetQuater
+            (int x, int y, int expected)
+        {
+            var actual = CalculationHelper.DetermineQuaterOfEnteredCoords(x, y);
+
+            Assert.AreEqual(expected, actual);
+        }
+
+        [TestCase(0, 1)]
+        [TestCase(1, 0)]
+        [TestCase(0, 0)]
+        public void DetermineQuaterOfEnteredCoords_WhenXOrYIsZero_ShouldThrowArgumentException
+            (int x, int y)
+        {
+            Assert.Throws<ArgumentException>(() =>
+            {
+                CalculationHelper.DetermineQuaterOfEnteredCoords(x, y);
+            });
+        }
+
+        [TestCase(0, 9, 5, 0, 5, 9)]
+        [TestCase(11, -4, 2, -4, 2, 11)]
+        [TestCase(-8, -12, -40, -40, -12, -8)]
+        [TestCase(0, 5, 0, 0, 0, 5)]
+        [TestCase(7, 0, -5, -5, 0, 7)]
+        public void PrintAscendingOrder_WhenABCPassed_ShoudPrintABCInAscendingOrder
+            (int a, int b, int c,
+             int expectedA, int expectedB, int expectedC)
+        {
+            (int a, int b, int c) actual = CalculationHelper.PrintAscendingOrder(a, b, c);
+            Assert.AreEqual(expectedA, actual.a);
+            Assert.AreEqual(expectedB, actual.b);
+            Assert.AreEqual(expectedC, actual.c);
+        }
+
+        [TestCase(2, 5, -3, 0.5d, -3)]
+        [TestCase(2, 6, 0, 0, -3)]
+        [TestCase(5, -6, 1, 1, 0.2d)]
+        public void GetRootsOfQuadraticEquation_WhenABCPassed_ShoudReturnRootsOfQuadraticEquation
+            (float a, float b, float c, double expectedX1, double expectedX2)
+        {
+            (double x1, double x2) actual = CalculationHelper.GetRootsOfQuadraticEquation(a, b, c);
+            Assert.AreEqual(expectedX1, actual.x1, 0.01);
+            Assert.AreEqual(expectedX2, actual.x2, 0.01);
+        }
+
+        [TestCase(4, 2, 2)]
+        [TestCase(5, 3, 3)]
+        public void GetRootsOfQuadraticEquation_WhenDiscriminantLessThenZero_ShoudThrowArgumentException
+            (float a, float b, float c)
+        {
+            Assert.Throws<ArgumentException>(() =>
+            {
+                CalculationHelper.GetRootsOfQuadraticEquation(a, b, c);
+            });
+        }
+
+        [TestCase(0, 2, 2)]
+        public void GetRootsOfQuadraticEquation_WhenAIsEqualToZero_ShoudThrowArgumentException
+            (float a, float b, float c)
+        {
+            Assert.Throws<ArgumentException>(() =>
+            {
+                CalculationHelper.GetRootsOfQuadraticEquation(a, b, c);
+            });
+        }
+
+        [TestCase(22, "TwentyTwo")]
```

Comment:
Ten test cases for exceptions 10-19
Ten test cases for each tens 20,30,40
Ten test cases for each ones 23,44,54